toolchain/lex/tokenized_buffer.h

```diff
@@ -322,6 +307,7 @@ class TokenizedBuffer : public Printable<TokenizedBuffer> {
                            SourceBuffer& source)
       : value_stores_(&value_stores), source_(&source) {}
 
+  auto FindLineIndexImpl(int32_t offset) const -> LineIndex;
```

Comment:
Why just `offset` here but `byte_offset` elsewhere? Comments could also help explain what this is an offset into.

Reply:
No reason, switched to `byte_offset`. Comment still useful?

Reply:
Thanks!

I still appreciate comments on functions, but understand if you're going to decline.

Reply:
Comment added (to the definition since this is supposed to be an implementation detail function). Happy to add / move as useful in follow up.